Makefile

```diff
@@ -173,12 +162,6 @@ cargo-fmt:
 check-benches:
 	cargo check --workspace --benches --features "$(TEST_FEATURES)"
 
-# Runs only the ef-test vectors.
-run-ef-tests:
-	rm -rf $(EF_TESTS)/.accessed_file_log.txt
-	cargo test --release -p ef_tests --features "ef_tests,$(EF_TEST_FEATURES)"
-	cargo test --release -p ef_tests --features "ef_tests,$(EF_TEST_FEATURES),fake_crypto"
-	./$(EF_TESTS)/check_all_files_accessed.py $(EF_TESTS)/.accessed_file_log.txt $(EF_TESTS)/consensus-spec-tests
 
 # Runs EF test vectors with nextest
 nextest-run-ef-tests:
```

Comment:
Maybe we could just call this `run-ef-tests` and similarly with other targets? i.e. remove `nextest` from the names, seeing as it is no longer differentiating the target from anything else